docs/shared/angular-tutorial/angular-monorepo.md

```diff
@@ -0,0 +1,1034 @@
+
+       Lint errors found in the listed files.
+
+
+    ✔  nx run orders:lint (1s)
+    ✔  nx run angular-store:lint (1s)
+    ✔  nx run angular-store-e2e:lint (689ms)
+    ✔  nx run inventory-e2e:lint (690ms)
+    ✔  nx run inventory:lint (858ms)
+    ✔  nx run shared-ui:lint (769ms)
+
+ ———————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+
+ >  NX   Ran target lint for 7 projects (3s)
+
+    ✔    6/7 succeeded [0 read from cache]
+
+    ✖    1/7 targets failed, including the following:
+         - nx run products:lint
+```
+
+If you have the ESLint plugin installed in your IDE you should immediately see an error:
+
+![ESLint module boundary error](/shared/images/tutorial-angular-standalone/angular-standalone-module-boundaries.png)
+
+Learn more about how to [enforce module boundaries](/core-features/enforce-module-boundaries).
+
+## Setting Up CI
+
+Without adequate tooling, CI times tend to grow exponentially with the size of the codebase. Nx helps decrease the average CI time with the [`affected` command](/concepts/affected) and Nx Cloud's [distributed caching](/core-features/remote-cache). Nx also [decreases the worst case CI time](/concepts/dte) with Nx Cloud's distributed task execution.
+
+To set up Nx Cloud run:
+
+```shell
+nx connect
+```
+
+And click the link provided. You'll need to follow the instructions on the website to sign up for your account.
+
+Then you can set up your CI with the following command:
+
+```shell
+nx generate ci-workflow --ci=github
+```
+
+{% callout type="note" title="Choose your CI provider" %}
+You can choose `github`, `circleci`, `azure`, `bitbucket-pipelines`, or `gitlab` for the `ci` flag.
+{% /callout %}
+
+This will create a default CI configuration that sets up Nx Cloud to [use distributed task execution](/core-features/distribute-task-execution). This automatically runs all tasks on separate machines in parallel wherever possible, without requiring you to manually coordinate copying the output from one machine to another.
+
+## Next Steps
+
+Here's some more things you can dive into next:
+
+- Learn more about the [underlying mental model of Nx](/concepts/mental-model)
+- Learn about popular generators such as [how to setup Tailwind](/recipes/angular/using-tailwind-css-with-angular-projects)
+- Learn how to [migrate your existing Angular CLI repo to Nx](/recipes/angular/migration/angular)
+- [Setup Storybook for our shared UI library](/recipes/storybook/overview-angular)
+- [Speed up CI: Run only tasks for project that got changed](/core-features/run-tasks#run-tasks-affected-by-a-pr)
+- [Speed up CI: Share your cache](/core-features/remote-cache)
+- [Speed up CI: Distribute your tasks across machines](/core-features/distribute-task-execution)
+
+Also, make sure you
+
+- [Join the Nx community Slack](https://go.nrwl.io/join-slack) to ask questions and find out the latest news about Nx.
```

Comment:
shouldn't be Slack, but Discord now => URL is: http://go.nx.dev/community

(can you fix that in the React monorepo one as well, I missed it there: https://nx.dev/getting-started/tutorials/react-monorepo-tutorial#next-steps)